### PR TITLE
Minor fixes to Mac OS X docs

### PR DESCRIPTION
**Pull request by TimDaub**

### CODE REVIEW

File: docs/server/source/appendices/docker-on-mac.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Install Docker for Mac.
 Pull the bigchaindb and other required docker images from docker hub.
 
 ```text
-docker pull bigchaindb/bigchaindb:master
+docker pull bigchaindb/bigchaindb:latest
```
Comment (TimDaub) [inline docs/server/source/appendices/docker-on-mac.md]: I was assuming "latest" is latest release?

Comment (muawiakh) [reply docs/server/source/appendices/docker-on-mac.md]: `latest` is the latest release i.e. built from 1.1 branch. Master is built from master codebase. I still am a bit unsure about the question :)

Comment (ttmc) [reply docs/server/source/appendices/docker-on-mac.md]: I guess you could write:

```text
docker pull bigchaindb/bigchaindb:TAG
```

and then explain that TAG is one of the tag options for the bigchaindb/bigchaindb image listed on Docker Hub at https://hub.docker.com/r/bigchaindb/bigchaindb/tags/ . The image with the `master` tag was built using the current code of the BigchainDB master branch and `latest` is the same as the highest-numbered release.

Comment (TimDaub) [reply docs/server/source/appendices/docker-on-mac.md]: > latest is the latest release i.e. built from 1.1 branch. Master is built from master codebase. I still am a bit unsure about the question :)

That's all I wanted to know. Thx

> I guess you could write:

As with pip, we want users to download the latest release and not master.
I'll keep it as `latest`, OK?

Comment (ttmc) [reply docs/server/source/appendices/docker-on-mac.md]: I guess there's a different docs page about using Docker for dev/test, so here it's probably fine if you tell them to use the `latest` tag

Comment (muawiakh) [reply docs/server/source/appendices/docker-on-mac.md]: 👍

Comment (TimDaub) [reply docs/server/source/appendices/docker-on-mac.md]: OK. So can you give your approval then to have this merged?

**Comment by muawiakh** [comment]
I have integrating the changes in my [PR](https://github.com/bigchaindb/bigchaindb/pull/1825), maybe once that is merged, we can abandon it or vice versa.

**Comment by ttmc** [comment]
I'm closing this PR because PR #1825 deleted the file `docker-on-mac.md` and that PR is now merged.

**Pull request closed without merging**